include/mbedtls/pk.h

```diff
@@ -388,7 +388,7 @@ int mbedtls_pk_setup(mbedtls_pk_context *ctx, const mbedtls_pk_info_t *info);
  */
 int mbedtls_pk_setup_opaque(mbedtls_pk_context *ctx,
                             const mbedtls_svc_key_id_t key);
-#endif /* MBEDTLS_USE_PSA_CRYPTO */
+#endif /* MBEDTLS_PSA_CRYPTO_CLIENT */
```

Comment:
Now that `MBEDTLS_PK_OPAQUE` is enabled whenever `MBEDTLS_PSA_CRYPTO_CLIENT` is enabled, we need to test it. We can't test client-only configurations, but we still need to change the guards on testing of `MBEDTLS_PK_OPAQUE` from `MBEDTLS_USE_PSA_CRYPTO` to `MBEDTLS_PSA_CRYPTO_C`.